app/public/js/location_report.js

```diff
@@ -6,7 +6,7 @@ function initMap() {
 	var map = new google.maps.Map(document.getElementById('map'), {
 	  zoom: 15,
 	  center: locationPoint,
-	  draggable: false,
+	  draggable: true,
```

Comment:
I see that all the issue's tasks were checked off.  Did you test each and every one of the subtasks?

I'll test tonight to see if this somehow updates location search radius and zoom levels too.

Also, the location report shouldn't be draggable.  Double click to zoom is fine but dragging the location report's map around doesn't seem useful and potentially confusing.  If you drag the centre of the map off the screen, you can forget where the location is that the report is dealing with.

Reply:
I tested and see that most of the subtasks are incomplete as expected by viewing the code changes.